.github/workflows/ci.yml

```diff
@@ -9,14 +9,15 @@ on:
 
 jobs:
   build-test-matrix:
-    name: ${{ matrix.jdk-distribution }}-${{ matrix.os }}
+    name: ${{ matrix.jdk-distribution }}-${{ matrix.jdk-version }}-${{ matrix.os }}
```

Comment:
I would recommend not using matrix builds for JDK testing, instead using the multi-JDK support provided by `jboss-parent`.

Reply:
Done.